lib/sgtn-client/api/translation.rb

```diff
@@ -52,7 +60,12 @@ def self.getStrings(component, locale)
 
         default = SgtnClient::Config.configurations.default
         if items.nil? || items["messages"] == nil
-          items = SgtnClient::Source.getSources(component, default)
+          items = {}
+          s = SgtnClient::Source.getSources(component, default)
+          key, value = s.first
+          items["component"] = key
```

Comment:
Why assign key to component?

Reply:
the key is the component name from YML source file.

Reply:
Change the names of var to more meaningful names to be able to understand easily.

Reply:
fixed